src/components/Tag/Tag.scss

```diff
@@ -43,17 +43,22 @@ $icon-size: rem(16px);
   &:hover,
   &:focus,
   &:active {
-    @include recolor-icon(color('ink'));
-    background: color('sky', 'dark');
+    @include recolor-icon(var(--p-icon-on-surface, color('ink')));
+    background: var(--p-neutral-action-hovered, color('sky', 'dark'));
   }
```

Comment:
This will mean that focus ends up getting the same style as hover, which doesn't follow the system. In the new system, focused elements have no change in background style, but do get the new focus ring applied. It would make sense to keep that consistent for this component, so we have a uniform focus state across components.

Reply:
Thanks for the explanation 🙏 It clears up a lot, I was wondering why there were so few focus states in the system.